### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by Andrew1995bono**
Ngx-pagination was added and customized per Figma mock-up.

### CODE REVIEW

File: src/app/components/all-products-components/all-products-content/all-products-content.component.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   templateUrl: './all-products-content.component.html',
-  styleUrls: ['./all-products-content.component.css']
+  styleUrls: ['./all-products-content.component.scss'],
+  encapsulation: ViewEncapsulation.None
 })
 export class AllProductsContentComponent implements OnInit {
 
-  @Input() products?: ProductCard[];
-
+  @Input() products?: any;
```
Comment (SStegniy) [inline src/app/components/all-products-components/all-products-content/all-products-content.component.ts:12]: Please, avoid any. You have ProductCard interface

File: src/app/components/all-products-components/all-products-content/all-products-content.component.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+  p: number = 3;
```
Comment (SStegniy) [inline src/app/components/all-products-components/all-products-content/all-products-content.component.ts:13]: Add access modifier for variable (public, private, protected)

**Review by SStegniy — APPROVED** [review]

File: src/app/components/all-products-components/all-products-content/all-products-content.component.ts (hunk 2 above)
Comment (Andrew1995bono) [reply src/app/components/all-products-components/all-products-content/all-products-content.component.ts:13]: fixed

File: src/app/components/all-products-components/all-products-content/all-products-content.component.ts (hunk 1 above)
Comment (Andrew1995bono) [reply src/app/components/all-products-components/all-products-content/all-products-content.component.ts:12]: +fixed

**Pull request merged**